test/PreAMMBatcher.test.ts

```diff
@@ -105,6 +105,29 @@ describe("PreAMMBatcher: Unit Tests", () => {
   it("DOMAIN_SEPARATOR is correct", async () => {
     expect(await batcher.DOMAIN_SEPARATOR()).to.equal(DOMAIN_SEPARATOR);
   });
+  describe("inverse()", () => {
```

Comment:
nit: Could you add spacing between tests.

Reply:
All done! I re-requested review